dracut/modules.d/90kiwi-repart/kiwi-repart-disk.sh

```diff
@@ -277,8 +287,10 @@ setup_debug
 # *and* the filesystem is brand new ;) by masking it.
 # "systemctl disable" does not work here, because it is event driven
 # More details: https://github.com/SUSE/kiwi/issues/1034
-info "disable systemd-fsck-root.service"
-systemctl mask systemd-fsck-root.service
+mask_fsck_root_service
+
+# make sure we unmask the fsck service
+trap unmask_fsck_root_service EXIT
```

Comment:
shouldn't this be the other way round? First trap, then mask?

I know it will not make a difference in practice, but still it looks wrong.

Reply:
Yes I think you are right